src/rastervision/labels/object_detection_labels.py

```diff
@@ -84,6 +84,16 @@ def get_subwindow(self, window, ioa_thresh=1.0):
     def get_boxes(self):
         return [Box.from_npbox(npbox) for npbox in self.boxlist.get()]
 
+    def get_intersection_boxes(self, other):
```

Comment:
I think you added this because you want to get the boxes in the global frame of reference and not the window/local frame of reference which is what `get_subwindow` does. I think it would be helpful to add a comment or PyDoc distinguishing what this does from `get_subwindow`.

Reply:
can do